src/ctl/ctl.c

```diff
@@ -348,31 +301,32 @@ static int ctl_exec_query_write(void *ctx, const umf_ctl_node_t *n,
  */
 static int ctl_exec_query_runnable(void *ctx, const umf_ctl_node_t *n,
                                    umf_ctl_query_source_t source, void *arg,
-                                   umf_ctl_index_utlist_t *indexes,
+                                   size_t size, umf_ctl_index_utlist_t *indexes,
                                    const char *extra_name,
                                    umf_ctl_query_type_t query_type) {
     (void)extra_name, (void)query_type;
     assert(n != NULL);
     assert(n->cb[CTL_QUERY_RUNNABLE] != NULL);
     assert(MAX_CTL_QUERY_TYPE != query_type);
-    return n->cb[CTL_QUERY_RUNNABLE](ctx, source, arg, indexes, NULL,
-                                     MAX_CTL_QUERY_TYPE);
+    return n->cb[CTL_QUERY_RUNNABLE](ctx, source, arg, size, indexes,
```

Comment:
extra name is used so remove (void)extra_name few lines above

Reply:
Done.